packages/vuetify/src/components/VSelect/VSelect.ts

```diff
@@ -604,6 +604,7 @@ export default baseMixins.extend<options>().extend({
       // listIndex change from menu
       if (this.isMenuActive && keyCode !== keyCodes.tab) {
         menu.changeListIndex(e)
+        this.$emit('update:listIndex', menu.listIndex)
```

Comment:
`this.$emit('update:list-index', menu.listIndex)`

Reply:
Thanks, I was thinking about this but just next to this I saw `update:searchInput` event and I was confused which approach is the correct one for events. So we are keeping kebab-case, right?

Reply:
i gotta ask john about the searchInput, i think that got overlooked in updating for 2.0 (even though newer multiword events are kebab). go kebab for now, worse case its an easy revert.

Reply:
https://vuejs.org/v2/guide/components-custom-events.html#Event-Names

https://github.com/vuejs/vue/issues/6428

Reply:
@KaelWD nothing to add :)

- Changed the event name to kebab case
- Updated the playground code to match